docs-src/concepts/what-is-an-update.md

```diff
@@ -49,3 +49,17 @@ An Update has four phases.
 4. **Completion.** The Update handler can return a result or a language-appropriate error/exception to indicate its completion.
    The Platform sends the Update outcome back to the original invoking entity as an Update response.
    A [WorkflowExecutionUpdateCompletedEvent](/references/events#workflowexecutionupdatecompletedevent) Event in the Workflow Execution Event History denotes the completion of an Update.
+
+:::note
+
+Workflow Updates are disabled by default.
+
+To enable `UpdateWorkflowExecution` API, set [frontend.enableUpdateWorkflowExecution](https://github.com/temporalio/temporal/blob/e96b529a72dd75635f06d9eca0682897f77bc104/common/dynamicconfig/constants.go#L267) dynamic config value to `true`.
```

Comment:
```suggestion
To enable the `UpdateWorkflowExecution` API, set [frontend.enableUpdateWorkflowExecution](https://github.com/temporalio/temporal/blob/e96b529a72dd75635f06d9eca0682897f77bc104/common/dynamicconfig/constants.go#L267) dynamic config value to `true`.
```

- Do we have a Dynamic config concept to link to?
- OK with linking to the code... but maybe the file in general instead of a commit ? https://github.com/temporalio/temporal/blob/main/common/dynamicconfig/constants.go